### PR TITLE
[CWS] load_module: Fix the "path_resolution_error" event when loaded_from_memory

### PR DESCRIPTION
**Pull request by spikat**
### What does this PR do?

Fix the issue when a "path_resolution_error" event was sent each time a module was loaded from memory.

### Describe how to test/QA your changes

1/ Create a rule like:
```
  - id: module_loaded_from_memory
    expression: load_module.name == "mod" && load_module.loaded_from_memory == true
```
2/ Create a tiny module just doing printk:
```
#include <linux/init.h>
#include <linux/module.h>
#include <linux/kernel.h>

MODULE_LICENSE("GPL");
MODULE_AUTHOR("EZ. Spi");
MODULE_DESCRIPTION("Just a testing module.");
MODULE_VERSION("0.42");

static int __init lkm_example_init(void) {
 printk(KERN_INFO "Hello, World!\n");
 return 0;
}

static void __exit lkm_example_exit(void) {
 printk(KERN_INFO "Goodbye, World!\n");
}

module_init(lkm_example_init);
module_exit(lkm_example_exit);
```
3/ and a tool to load it:
```
#define _GNU_SOURCE
#include <fcntl.h>
#include <sys/syscall.h>
#include <sys/types.h>
#include <stdlib.h>
#include <stdio.h>
#include <sys/types.h>
#include <sys/stat.h>
#include <fcntl.h>
#include <errno.h>
#include <unistd.h>
#include <string.h>

#define init_module(module_image, len, param_values) syscall(__NR_init_module, module_image, len, param_values)

int main(int ac, char** av)
{
  int fd = open("./mod.ko", O_RDONLY);
  if (fd < 0) {
    puts("Open failed.\n");
    return (errno);
  }
  char buff[1024*512];
  int nbread = read(fd, buff, sizeof(buff));
  close(fd);
  if (init_module(buff, nbread, "") < 0) {
    printf("init_module failed: %s\n", strerror(errno));
    return (errno);
  }
  return (0);
}
```
4/ Compile and run both

### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] The appropriate `team/..` label has been applied, if known.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

File: pkg/security/probe/serializers.go [hunk 1]
```diff
@@ -800,8 +800,10 @@ func NewEventSerializer(event *Event) *EventSerializer {
 		s.PTraceEventSerializer = newPTraceEventSerializer(event)
 	case model.LoadModuleEventType:
 		s.EventContextSerializer.Outcome = serializeSyscallRetval(event.LoadModule.Retval)
-		s.FileEventSerializer = &FileEventSerializer{
-			FileSerializer: *newFileSerializer(&event.LoadModule.File, event),
+		if event.LoadModule.LoadedFromMemory == false {
```
Comment (paulcacheux) [inline pkg/security/probe/serializers.go:803]: Let's simplify that

File: pkg/ebpf/bytecode/runtime/conntrack.go [hunk 2]
```diff
@@ -4,4 +4,4 @@
 
 package runtime
 
-var Conntrack = NewRuntimeAsset("conntrack.c", "a72a9cf2b321f8825b1733b72e264a5df37c1aa280f0ff65c3900a5df84c993c")
```
Comment (paulcacheux) [inline pkg/ebpf/bytecode/runtime/conntrack.go:7]: We usually don't commit those changes when there is no edit to a related C file. They generally change when you run `inv -e system-probe.build` and the hash may depends on the installed clang version (which is different depending on teams/users it seems).

**Review by paulcacheux — CHANGES_REQUESTED** [review]

**Review by paulcacheux — APPROVED** [review]

File: pkg/security/probe/serializers.go (hunk 1 above)
Comment (spikat) [reply pkg/security/probe/serializers.go]: resolved on last commit

File: pkg/ebpf/bytecode/runtime/conntrack.go (hunk 2 above)
Comment (spikat) [reply pkg/ebpf/bytecode/runtime/conntrack.go]: resolved on last commit

**Pull request merged**